packages/prettier-config/.prettierrc.json

```diff
@@ -4,5 +4,13 @@
   "singleQuote": true,
   "tabWidth": 2,
   "trailingComma": "all",
-  "useTabs": false
+  "useTabs": false,
+  "overrides": [
+    {
+      "files": "*.svg",
```

Comment:
Looks like the `files` key is typed as an array of string: https://github.com/prettier/prettier-vscode/wiki/Custom-File-Extension-Configuration.
As SVG is not a "real" file format, we may want to add `*.xml` in this config if we want to use _html_ parser for these kind of files.